python/dalex/dalex/_explainer/object.py

```diff
@@ -298,7 +298,10 @@ def model_profile(self,
         types = ('partial', 'accumulated', 'conditional')
         type = check_method_type(type, types)
 
-        N = min(N, self.data.shape[0])
+        if N is None:
+            N = self.data.shape[0]
+        else:
+            N = min(N, self.data.shape[0])
```

Comment:
can we unify that?
https://github.com/ModelOriented/DALEX/blob/44d722a6dce6d6569752d1fca01c606b7d69b8b5/python/dalex/dalex/dataset_level/_variable_importance/utils.py#L38-L43

Reply:
unified